src/modules/drive/RenameInput.jsx

```diff
@@ -14,7 +14,7 @@ import FilenameInput from '@/modules/filelist/FilenameInput'
 // If we set the _rev then CozyClient tries to update. Else
 // it tries to create
 const updateFileNameQuery = async (client, file, newName) => {
-  return client.save({
+  return client.collection('io.cozy.files', { driveId: file.driveId }).update({
```

Comment:
I wonder what the comment means or if by update from save to update we change the behavior 🤔

Reply:
Can you just double check that this method is not used for something else than renaming and there is no regression?